app/models/course.rb

```diff
@@ -79,4 +83,12 @@ def open_for_applications?
   def has_vacancies?
     site_statuses.with_vacancies.any?
   end
+
+  def qualifications
+    Qualifications.new(
```

Comment:
We may need to be aware of performance issues in relation to the below to lines

```
      is_pgde: self.in?(Course.pgde),
      is_fe: subjects.further_education.any?,
```

Reply:
Agreed – I think we shouldn't merge this branch until we've performance-tested it with the frontend, real-world worst-case usage.